platform_versioned_docs/version-24.1/data_studios/index.mdx

```diff
@@ -0,0 +1,114 @@
+
+## Requirements
+
+Before you get started, ensure you have the following:
+
+- Valid credentials to access your cloud storage data resources.
+- At least the **Maintain** role set of permissions.
+- A compute environment with sufficient resources. This is highly dependent on the volume of data you wish to process, but we recommended at least 2 CPUs allocated with 8192 MB of memory. See [AWS Batch](../compute-envs/aws-batch.mdx) for more information about compute environment configuration.
+- [Data Explorer](../data/data-explorer.mdx) is enabled.
+
+:::note
+Currently, Data Studios only supports AWS Batch compute environments that **do not** have Fargate enabled.
+:::
+
+## Container image templates
+
+Data Studios provides four container image templates: JupyterLab, RStudio Server, Visual Studio Code, and Xpra. The image templates install a very limited number of packages when the data studio session container is built. Users have permission to install additional packages as needed.
+
+In addition, you can create your own container image templates. For more information, see [Custom environments][custom-envs].
+
+**JupyterLab**
+
+The default user is the `root` account. The following [conda-forge](https://conda-forge.org/) packages are available by default:
+
+- `altair`
+- `ipywidgets`
+- `itables`
+- `jedi-language-server`
+- `jupyter-collaboration`
+- `jupyter-dash`
+- `jupyterlab-git`
+- `jupyterlab`
+- `jupytext`
+- `nb_black`
+- `pandas[all]`
+- `pip`
+- `plotly`
+- `python`
+- `qgrid`
+- `r-ggplot2`
+- `scikit-learn`
+- `seaborn[stats]`
+- `statsmodels`
+
+To install additional Python packages, execute `!pip install <packagename>` commands in your notebook environment. Additional system-level packages can be installed in a terminal window using `apt install <packagename>`.
+
+**RStudio Server**
+
+The default user is the `rstudio` account and has [sudo](https://en.wikipedia.org/wiki/Sudo) privileges. To install R packages, execute `install.packages("<packagename>")` commands in your notebook environment. Additional system-level packages can be installed in a terminal window using `sudo apt install <packagename>`.
+
+**Visual Studio Code**
+
+[Visual Studio Code][vsc] is an integrated development environment (IDE) that supports many programming languages. The default user is the `root` account. To install extensions, select **Extensions**. Additional system-level packages can be installed in a terminal window using `apt install <packagename>`.
+
+**Xpra**
```

Comment:
Version: `6.2.0-r2-1` (maybe r2-1 could be omitted since it's from the os package manager)